Modules/_decimal/_decimal.c

```diff
@@ -1566,17 +1567,27 @@ init_extended_context(PyObject *v)
 }
 
 /* Factory function for creating IEEE interchange format contexts */
+
+/*[clinic input]
+_decimal.Decimal.IEEEContext
```

Comment:
Isn't this a global function?
```suggestion
_decimal.IEEEContext
```